generic-ec-curves/Cargo.toml

```diff
@@ -11,30 +11,28 @@ repository = "https://github.com/dfns/generic-ec"
 [dependencies]
 generic-ec-core = { version = "0.2", path = "../generic-ec-core", default-features = false }
 
-subtle = { version = "2.4", default-features = false }
-rand_core = { version = "0.6", default-features = false }
-zeroize = { version = "1", default-features = false, features = ["zeroize_derive"] }
+subtle.workspace = true
+rand_core.workspace = true
+zeroize = { workspace = true, features = ["zeroize_derive"] }
 
 elliptic-curve = { version = "0.13", default-features = false, features = ["sec1", "hash2curve"], optional = true }
 k256 = { version = "0.13", optional = true, default-features = false, features = ["hash2curve"] }
 p256 = { version = "0.13", optional = true, default-features = false, features = ["hash2curve"] }
-sha2 = { version = "0.10", default-features = false, optional = true }
+sha2 = { workspace = true, optional = true }
 stark-curve = { version = "0.1", default-features = false, optional = true }
 
 group = { version = "0.13", default-features = false, optional = true }
 
 [dependencies.curve25519]
-package = "curve25519-dalek"
-version = "4"
-default-features = false
+workspace = true
 features = ["group", "zeroize", "rand_core", "precomputed-tables"]
 optional = true
```

Comment:
Oops, I meant this line is what confuses the script. And it's already removed, so no problems. Still, why not rewrite it in `{}` style? It is my personal nitpick though, I just don't like this part of toml syntax usage

Reply:
I like it though, it's neat when `{}` style results into too long lines.

Reply:
or did you already get used to >200 lines :D

Reply:
`curve25519 = { workspace = true, features = ["group", "zeroize", "rand_core", "precomputed-tables"], optional = true }`

nah, it's certainly less readable